package.json

```diff
@@ -81,19 +81,19 @@
     "lint:production:stylelint": "tsc --noCheck --outDir ./dist && stylelint '**/*.styles.ts' --custom-syntax postcss-lit --no-color",
     "test": "per-env",
     "test:development": "pnpm test:development:web-test-runner",
-    "test:development:playwright": "playwright test --config ./src/playwright/playwright.config.ts --quiet --ui-port 6007 --update-snapshots --update-source-method=overwrite",
-    "test:development:playwright:update": "rimraf ./src/aria-snapshots && playwright test --config ./src/playwright/playwright.config.ts --quiet --project aria --update-snapshots --update-source-method=overwrite --workers 100%",
+    "test:development:playwright": "npm-run-all --parallel test:development:playwright:coverage test:development:playwright:ui",
+    "test:development:playwright:coverage": "http-server ./dist/playwright/coverage-report --port 6008 --silent",
```

Comment:
`http-server` is back! 

Playwright is able to serve our coverage report. But it would open a browser tab with the report every time this script is run. If you're okay with that, let me know. I'll replace `http-server` with the equivalent Playwright command.

Reply:
- `:6006`: Storybook
- `:6007`: Playwright: 
- `:6008`: Coverage Report